### ISSUE TITLE
 Bootstrap 5 Scrollspy -  question 

### ISSUE THREAD

**Issue by devwebbuilder** [twbs/bootstrap#33328]
Hello,
when I am watching and learn the section Bootstrap from Udemy, in Scrollspy(component) there is a "small" problem.
(It may be something simple but I was educated for 2 hours and I did not get an edge).
So by the Rob's video about v4-alpha bootstrap I did it [(scrollspy(component) builded from the [course)]. --> [v4bootstrap scrollspy](http://webdevbuilder.tech/5-bootstrap/bootstrap.html)
I am trying to code it from begin and use bootstrap 5 and the component(scrollspy) do not work correctly. (scrollspy(component) builded from begin in bootstrap  5 but not working correctly). --> [bootstrap 5 scrollspy](http://webdevbuilder.tech/5-bootstrap/bootstrap5.html)
So I also try the Example in navbar from bootstrap 5 and not working correctly too! (scrollspy(component) Example in navbar builded from begin in bootstrap  5 but not working correctly). --> [bootstrap 5 example scrollspy](http://webdevbuilder.tech/5-bootstrap/bootstrap-5-example.html)
So where is the problem?
What am I doing wrong?
I believe that I have entered cdn libraries from bootstrap css and javascript correct.
Thank you
Sincerely

**Comment by mdo** [comment]
Check the markup and compare against our docs again—our examples show it working right now. v4 and v5 have some differences between the two across HTML, CSS, and JS.

**Comment by devwebbuilder** [comment]
Thank you about immediate response and tips.
So I check it and try again and I fixed this.
In scrollspy from bootstrap 5 version I have added in navbar element the pacement :  sticky-top to fix it and working correctly, and also remove the div container with content and attributes and add them only in the body.
Take a look about scrollspy bootstrap5 [here](http://webdevbuilder.tech/5-bootstrap/bootstrap5.html)
About bootstrap 5 example scrollspy I have added also like previous example in navbar element the pacement :  sticky-top to fix it and working correctly.
After I have added the position-relative class to the body and an attribute data-bs-offset for calculating position of scroll and working correctly.
Take a look about bootstrap 5 example scrollspy [here](http://webdevbuilder.tech/5-bootstrap/bootstrap-5-example.html)
Sincerely...
devwebbuilder
